cluster-autoscaler/cloudprovider/aws/aws_manager_test.go

```diff
@@ -48,6 +48,31 @@ func TestBuildGenericLabels(t *testing.T) {
 	assert.Equal(t, cloudprovider.DefaultOS, labels[kubeletapis.LabelOS])
 }
 
+func TestExtractAllocatableResourcesFromAsg(t *testing.T) {
+	tags := []*autoscaling.TagDescription{
+		{
+			Key:   aws.String("k8s.io/cluster-autoscaler/node-template/resources/cpu"),
+			Value: aws.String("100m"),
+		},
+		{
+			Key:   aws.String("k8s.io/cluster-autoscaler/node-template/resources/memory"),
+			Value: aws.String("100M"),
+		},
+		{
+			Key:   aws.String("k8s.io/cluster-autoscaler/node-template/resources/ephemeral-storage"),
+			Value: aws.String("20G"),
+		},
+	}
```

Comment:
Did you considered using table tests for some different values for example for cpu case "2"?